### ISSUE TITLE
[HOLD for payment 2024-01-26] [HOLD for payment 2024-01-25] [$500] Create new command for Payment cancelling in "Paid Elsewhere" reports

### ISSUE THREAD

**Issue by Gonals** [Expensify/App#32426]
**Problem**
Currently, we have no way to revert an existing payment in NewDot. This is part of our core functionality and something we should be able to do.

**Solution**
Let's add that functionality as discussed [here](https://github.com/Expensify/Expensify/issues/327308). We'll start by adding it for "Paid Elsewhere" reports, as it is simpler and will give us the core structure of the command and the UI, and move to "Paid in Expensify" reports in a [follow-up issue](https://github.com/Expensify/App/issues/32427).
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01c0fd3da0a160792d</li>
        <li>Upwork Job ID: 1731661662582071296</li>
        <li>Last Price Increase: 2024-01-05</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by mountiny** [comment]
[The PR](https://github.com/Expensify/App/pull/32674) has caused numerous deploy blockers in App:
- https://github.com/Expensify/App/issues/33507
- https://github.com/Expensify/App/issues/33513
- https://github.com/Expensify/App/issues/33540
- https://github.com/Expensify/App/issues/33508
- https://github.com/Expensify/App/issues/33509

Some might be considered as not a blockers but I think given the number of these and that the author is ooo I will revert the PR and we can just fix this in the second attempt

**Comment by michaelhaxhiu** [comment]
Seems we just need to pay $500 flat fee for PR review(s) to @narefyev91.

**Comment by michaelhaxhiu** [comment]
Using automation to create the new upwork job to pay Nicolay

**Comment by michaelhaxhiu** [comment]
Can we close this GH?

**Comment by mountiny** [comment]
@michaelhaxhiu We had to revert the PR so I am not sure if we put it back in place, I would wait for @Gonals to confirm

@narefyev91 is agency worker nothing to pay there

**Comment by michaelhaxhiu** [comment]
Roger that, I'll wait on Alberto's confirmation.

And totally makes sense re: agency worker, I spaced out 🛸 🤦

**Comment by Gonals** [comment]
It is in review! https://github.com/Expensify/App/pull/33836

**Comment by Gonals** [comment]
Man, this is being more of a pain than expected. Bringing some discussions to Slack

**Comment by zanyrenney** [comment]
@michaelhaxhiu please handle the payment when you get a chance!

**Comment by michaelhaxhiu** [comment]
There is no payment here! And no need for regression steps IMO (it's not a regression from a bug, it's a new feature).

**Comment by mallenexpensify** [comment]
@Ollyws sorry about that, glad you caught it! 
can you please accept the job and reply here once you have? 
https://www.upwork.com/jobs/~01f578768b7177142d

**Comment by mallenexpensify** [comment]
@Ollyws you were asked to review in January so we need to pay via Upwork
https://github.com/Expensify/App/pull/33836#event-11370487527

Please rescind your NewDot request.  It's best practice to wait for the BZ payment summary before requesting funds via NewDot. Thx

**Comment by mallenexpensify** [comment]
Contributor: @Ollyws  paid $500 via Upwork

Apologies again for the lonnnnng delay here @Ollyws
